.github/workflows/pr.yml

```diff
@@ -35,123 +35,6 @@ jobs:
       - name: TypeDoc
         uses: ./.github/actions/build-docs
 
-  run-integration-tests:
```

Comment:
This was part of the integration tests work which is also included here. This block of logic was moved to `testing.yml` so it could be more easily reused.